src/misc/lv_text.c

```diff
@@ -781,8 +781,8 @@ static uint32_t lv_text_iso8859_1_prev(const char * txt, uint32_t * i)
 
 /**
  * Convert a character index (in an ISO8859-1 text) to byte index.
- * E.g. in "AÁRT" index of 'R' is 2th char but start at byte 3 because 'Á' is 2 bytes long
- * @param txt a '\0' terminated UTF-8 string
+ * The iso8859-1 encoding is compatible with ASCII so the indices of characters is the same as the indices of bytes.
```

Comment:
ISO8859-1 Make sure to write it in uppercase.

Reply:
Great!